config/initializers/google_maps_config.rb

```diff
@@ -0,0 +1 @@
+Google::Maps.api_key = ENV['GOOGLE_MAPS_API_KEY'] || nil
```

Comment:
`ENV['GOOGLE_MAPS_API_KEY']` _should_ return `nil` if it isn't set, so you might not need the `|| nil` here.